client/web/web.go

```diff
@@ -397,7 +400,7 @@ func (s *Server) authorizeRequest(w http.ResponseWriter, r *http.Request) (ok bo
 			// All requests must be made over tailscale.
 			http.Error(w, "must access over tailscale", http.StatusUnauthorized)
 			return false
-		case r.URL.Path == "/api/data" && r.Method == httpm.GET:
+		case (r.URL.Path == "/api/data" || r.URL.Path == "/api/serve/items") && r.Method == httpm.GET: // TODO: maybe allow all GET?
```

Comment:
For this TODO: maybe safer to continue this pattern as we have it? Not sure if we have anything now that we care too too much about conditionally hiding (I think this would currently only affect `GET /exit-nodes`?) but could see us accidentally exposing when we don't want to if we allow all `GET` by default.